srcpkgs/walk/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'walk'
+pkgname=walk
+version=20190920
+revision=1
+_commit=70f7a8c104acf109ae810a9b34eb8e8b92b4d27d
+build_style="gnu-makefile"
+wrksrc="${pkgname}-${_commit}"
+short_desc="walk and sor utility, collectively replacing find"
+maintainer="orphaned"
```

Comment:
Why is this already orphaned?

Reply:
Do you want maintainership?

Reply:
Generally if you make the PR to add it, you be willing to maintain it.